detect/app.go

```diff
@@ -63,10 +63,21 @@ func GetAppNameFromGitRemote(directory string, remoteName string) (string, error
 	return "", errgo.Newf("[detect] Scalingo Git remote hasn't been found")
 }
 
+// Returns the remote name specified in command flags
```

Comment:
Just a picky comment: in Go, function comments start with the function name

```suggestion
// RemoteNameFromFlags returns the remote name specified in command flags
```

This is for a prettier output in the Go doc. For example: https://pkg.go.dev/github.com/Scalingo/go-scalingo/v5#NewStaticTokenGenerator